### PR TITLE
#12 Conventions - Contributing.md was updated

### PR DESCRIPTION
**Pull request by regelkraut**
according to the issue description

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -141,7 +167,19 @@ nullFlavor
 
 ## Additional Information
 
-- TBD (add mail addresses and contact information)
+### Documentation of decisions on issues
+HL7 International is documenting the decision it makes on issue in it's ticket tracker tool gforge. For HL7 Austria we've decided to use their way of documenting decisions for our github issues as well. Since github issue don't provide any forms to fill out certain attributes for an issue it was decided to document the decision criteria as a comment for each issue with the following attributes:
```
Comment (aschuler84) [inline CONTRIBUTING.md]: makes on issues << plural

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -141,7 +167,19 @@ nullFlavor
...
+- **Motion:** Exact wording of the motion which will be moved.
+- **Change Type:** describes the severity of the proposed change -> HL7 International categories (Any, None, Non-substantive, Compatible-substantive, Non-compatible)
+- **(Ballot) Resolution:** final resolution -> HL7 International (ballot) categories (Persuasive, Persuasive with Mod, ...)
+- **Mover/Seconder: For-Against-Abstain:**
+
+### Information if an issue is derived from a Ballot
+Since github issue cannot be created automatically out of HL7 Austria Ballot results, it was decided to document the Ballot information as a comment for each relevant issue with the following attributes:
```
Comment (aschuler84) [inline CONTRIBUTING.md]: issues << plural

File: CONTRIBUTING.md (hunk 1 above)
Comment (aschuler84) [inline CONTRIBUTING.md]: it makes on issues <<- plural

Comment (aschuler84) [reply CONTRIBUTING.md]: since GitHub issues <<- plural

File: CONTRIBUTING.md (hunk 2 above)
Comment (regelkraut) [reply CONTRIBUTING.md]: typo fixed an pushed

File: CONTRIBUTING.md (hunk 1 above)
Comment (regelkraut) [reply CONTRIBUTING.md]: typos fixed an pushed

**Pull request merged**